sdks/java/extensions/sql/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
   compile "com.alibaba:fastjson:1.2.68"
   compile "org.codehaus.janino:janino:3.0.11"
   compile "org.codehaus.janino:commons-compiler:3.0.11"
+  compile project(path: ":runners:google-cloud-dataflow-java")
```

Comment:
Do you need to change sql module's gradle file? The benchmarking tool is built on top of SQL, so SQL is just a library, right?

Reply:
Yes, will remove this